app/config/config.yml.dist

```diff
@@ -460,3 +460,5 @@ compatibility:
     # Response methods cannot be used on TemplateView objects.
     # Setting this value to false is deprecated.
     template_view: true
+    # Allows specifying a newer version of the setcontent parser
+    setcontent_version: 1
```

Comment:
What if we end up with a version 4 in the short term? Numerical comparison is easier, correct?

Mind you, wouldn't this be more suitable as a `compatibility` key?